feat(Applications): New applications documentation and taxonomy update (#1718)
bonitasoft/bonita-doc: modules/ROOT/pages/bonita-applications-deep-dive.adoc

= Applications deep dive 
:description: This page explains how Bonita runs applications and presents the Bonita applications natively embedded in Bonita Runtime

{description}

[.card-section]
== How it works in Bonita Runtime

[.card.card-index]
--
xref:bonita-applications-interface-overview.adoc[[.card-title]#Overview# [.card-body.card-content-overflow]#pass:q[How applications are run by Bonita]#]
--

[.card.card-index]
--
xref:cache-configuration-and-policy.adoc[[.card-title]#Cache management# [.card-body.card-content-overflow]#pass:q[Cache policies and how to configure the caching mechanism]#]
--

[.card.card-index]
--
xref:log-in-and-log-out.adoc[[.card-title]#Login mechanism# [.card-body.card-content-overflow]#pass:q[How to get users to log into an application]#]
--

[.card-section]
== Bonita applications
Bonitasoft developers develop Bonita using Bonita. +
This is why Bonita comes with four embedded applications, with different status: 

* Core applications. They are critical to the good operation of Bonita in a Production environment. They can be customized but they cannot be deleted:
  ** Application Directory 
  ** Super Administrator

> * Generic applications. They are fully functional, can be customized, but can be deleted if not useful in a Production environment:

Reviewer: ```suggestion
* Generic applications. They are fully functional, can be customized, but can also be deleted if not useful in a Production environment:
```

  ** Administrator
  ** User
  
[.card.card-index]
--

> xref:application-directory-application.adoc[[.card-title]#Application directory# [.card-body.card-content-overflow]#pass:q[The logged user's catalog of applications... and more]#]

Reviewer: ```suggestion
xref:application-directory.adoc[[.card-title]#Application directory# [.card-body.card-content-overflow]#pass:q[The logged user's catalog of applications... and more]#]
```

--

[.card.card-index]
--

> xref:super-administrator-application.adoc[[.card-title]#Super Administrator# [.card-body.card-content-overflow]#pass:q[The tool to bootstrap a Bonita Runtime, perform maintenance actions, and update the BDM]#]

Reviewer: ```suggestion
xref:super-administrator-application.adoc[[.card-title]#Super Administrator# [.card-body.card-content-overflow]#pass:q[The tool to bootstrap a Bonita Runtime, perform maintenance operations, and update the BDM]#]
```

--

[.card.card-index]
--

> xref:administrator-application.adoc[[.card-title]#Administrator# [.card-body.card-content-overflow]#pass:q[A tool to manage an automation project in production (BDM excepted)]#]

Reviewer: "manage an automation project in production" It is not clear to me. It may be me but when I read this I am not expecting the admin app but maybe a project management tool to help building the apps

Author: Yep. I can definitely understand. Trying to find something clearer.

Author: "Supervise and update automation projects in production (BDM excepted)"

Author: (So I also removed "the tool" for the super admin app. Afraid it is going to exceed the 3 rows we have for description on the cards.)

Reviewer: :+1:

--

[.card.card-index]
--

> xref:user-application.adoc[[.card-title]#User# [.card-body.card-content-overflow]#pass:q[A tool for actual people making processes alive, centered around a generic tasklist]#]

Reviewer: "making processes alive" I think about Dave when I reade this.
maybe
```suggestion
xref:user-application.adoc[[.card-title]#User# [.card-body.card-content-overflow]#pass:q[A tool for actual people using the processes, centered around a generic tasklist]#]
```
?

Author: I will go for "executing the processes". What they use is the applications, to me.

Reviewer: :+1:

Author: "Start process instances and execute tasks using a generic tasklist"

Author: done

--

[.card-section]
== Related topics

Learn how to create your own Custom Applications.

[.card.card-index]
--

> xref:custom-applications-index.adoc[[.card-title]#Custom applications# [.card-body.card-content-overflow]#pass:q[What it is made of, and who creates what when]#]

Reviewer: not sure but I read it better with a coma
```suggestion
xref:custom-applications-index.adoc[[.card-title]#Custom applications# [.card-body.card-content-overflow]#pass:q[What it is made of, and who creates what, when]#]
```
